docs/Ariba-batchsource.md

```diff
@@ -18,6 +18,7 @@ You can configure the following properties for the Ariba.
 
 **Reference Name:** Name used to uniquely identify this source for lineage,
 annotating metadata, etc.  
+**Token Endpoint (M)**:Token path of the Ariba API.
```

Comment:
Please be consistent with the formatting.

Reply:
done the changes